jerry-core/ecma/builtin-objects/ecma-builtin-json.cpp

```diff
@@ -1130,13 +1130,13 @@ ecma_builtin_json_stringify (ecma_value_t this_arg __attr_unused___, /**< 'this'
 
       for (uint32_t i = 0; i < 10; i++)
       {
-        space_buff[i] = zt_string_buff[i];
+        space_buff[i] = string_buff[i];
       }
```

Comment:
There is a condition above
```
if (num_of_chars < 10)
{
  context_p.gap_str_p = ecma_copy_or_ref_ecma_string (space_str_p);
}
else
{
  ...
}
```
If the number of characters are bigger than 10, the first 10 characters will be used.
If the number of characters are less than 10, all the characters will be used.